src/columnstore_handler.cpp

```diff
@@ -218,69 +218,67 @@ bool columnstore_scan_sample_next_tuple(TableScanDesc scan, struct SampleScanSta
     elog(ERROR, "columnstore_scan_sample_next_tuple not implemented");
 }
 
-const TableAmRoutine columnstore_routine = {
-    T_TableAmRoutine,
-
-    columnstore_slot_callbacks,
-
-    columnstore_scan_begin,
-    columnstore_scan_end,
-    columnstore_scan_rescan,
-    columnstore_scan_getnextslot,
-
-    NULL /*scan_set_tidrange*/,
-    NULL /*scan_getnextslot_tidrange*/,
-
-    columnstore_parallelscan_estimate,
-    columnstore_parallelscan_initialize,
-    columnstore_parallelscan_reinitialize,
-
-    columnstore_index_fetch_begin,
-    columnstore_index_fetch_reset,
-    columnstore_index_fetch_end,
-    columnstore_index_fetch_tuple,
-
-    columnstore_tuple_fetch_row_version,
-    columnstore_tuple_tid_valid,
-    columnstore_tuple_get_latest_tid,
-    columnstore_tuple_satisfies_snapshot,
-    columnstore_index_delete_tuples,
-
-    columnstore_tuple_insert,
-    columnstore_tuple_insert_speculative,
-    columnstore_tuple_complete_speculative,
-    columnstore_multi_insert,
-    columnstore_tuple_delete,
-    columnstore_tuple_update,
-    columnstore_tuple_lock,
-    NULL /*finish_bulk_insert*/,
+const TableAmRoutine columnstore_routine = {T_TableAmRoutine,
```

Comment:
`make format` does all these changes, no functionality change

Reply:
There seems to be something wrong with your local setup. In particular, we are not supposed to re-format `src/pgduckdb/pgduckdb_background_worker.cpp` below
Could you check if you still hit this after using our dev container?